Makefile

```diff
@@ -1503,6 +1525,7 @@ clean::
 	rm -f runtime/domain_state.inc
 	rm -rf $(DEPDIR)
 	rm -f stdlib/libcamlrun.a stdlib/libcamlrun.lib
+	rm -f $(winpthreads_OBJECTS)
```

Comment:
I don't think this is necessary there's a wildcard pattern for objects above?

I'm also slightly worried that it means an `rm -f` is executed on Unix, even if that's no-op!!